Completed code

Njolf opened the pull request:
uploaded code for the first 3 boxes - Salience, Wason & Peltzman. issues #14 , #13 & #11

Njolf committed:
upload code for the peltzman box

Njolf committed:
upload complete code for wason box

Njolf committed:
upload code for Salience box

atodorov commented on Salience/Salience.ino:
```diff
@@ -0,0 +1,85 @@
+#include <Encoder.h>
```
The directory for this file should be `Salience_bias_box/src`

Njolf replied:
got it, will change that

atodorov commented on Peltzman_effect_box/src/Peltzman.ino:
```diff
@@ -0,0 +1,441 @@
+#include <FastLED.h>
```
In this directory there are files `pseudocertainty_main.ino` and `shift_rand.ino`. Are they related or should they be deleted ? 

If there is only 1 source file per box how about naming the file `main.ino` instead ?

Njolf replied:
pseudocertainty_main.ino is the old & incomplete version, i'll remove it. as for shift_rand.ino, i'm not exactly sure what it's meant for, but it's not used elsewhere. i will rename the proper .ino files into "main.ino" if necessary, but the arduino IDE will make a fuss about it if you try opening them (it will request a separate "main" folder to put "main.ino" inside, otherwise it won't open it).

atodorov replied:
OK then. Please remove the old files and then use names which the Arduino IDE will like or adjust the dir structure to what the IDE needs (I don't know how that IDE actually works). Just keep it consistent for all directories.

Njolf replied:
i've renamed the files to src.ino (so they can fit into the src folders without issues). also, i've removed the old files and i've adjusted the directory for the Salience bias box.

atodorov commented on Wason_2-4-6_box/src/Wason.ino:
```diff
@@ -0,0 +1,743 @@
+#include "SevenSegmentTM1637.h"
```
There is `wason.ino` (lower case) in this directory already. How is it related to the new file ?

Njolf replied:
it's the incomplete version that i didn't remove (the difference between lowercase and uppercase is because of some IDE stuff in my directories, nothing significant).

atodorov reviewed (changes requested):
overall looks good, minor changes requested

atodorov commented:
For each individual commit, can you also add `Refs #<issue number>` to the commit log and reference the issues related to each particular box.

Njolf committed:
Delete pseudocertainty_main.ino (old)

Njolf committed:
Delete shift_rand.ino (unused)

Njolf committed:
Rename peltzman.ino to main.ino. Refs  kiwitcms#14

Njolf committed:
change directory for Salience bias box. Refs kiwitcms#13

Njolf committed:
cleaning up the directories. Refs kiwitcms#13

Njolf committed:
Delete main.ino

Njolf committed:
upload proper directory for Salience bias box. Refs kiwitcms#13

Njolf committed:
Rename main.ino to src.ino. Refs kiwitcms#14

i did it so there's no conflicts with the arduino IDE whilst not having too many subfolders.

Njolf committed:
Delete wason.ino (old)

Njolf committed:
Rename Wason.ino to src.ino. Refs kiwitcms#11

renamed it due to the arduino IDE.

Njolf committed:
Merge pull request #1 from Njolf/temp

Temp
